test/OracleTest.sol

```diff
@@ -334,84 +312,11 @@ contract OracleTest is Test {
-
-    //     vm.startPrank(bidder);
-    //     bidderRegistry.prepay{value: 250 ether }();
-    //     vm.stopPrank();
-
-    //     vm.startPrank(provider);
-    //     providerRegistry.registerAndStake{value: 250 ether}();
-    //     vm.stopPrank();
-
-    //     uint256 ogStake = providerRegistry.checkStake(provider);
-
-    //     string memory commitedTxn = string(abi.encodePacked(keccak256("0xSlash")));
-    //     constructAndStoreCommitment(bid, blockNumber, commitedTxn, bidderPk, providerPk);
-    //     vm.prank(address(0x6d503Fd50142C7C469C7c6B64794B55bfa6883f3));
-    //     oracle.addBuilderAddress("kartik builder", provider);
-    //     vm.expectEmit(true, true, false, true);
-    //     emit BlockDataReceived(txnList, blockNumber, blockBuilderName);
-    //     oracle.receiveBlockData(txnList, blockNumber, blockBuilderName);
-
-    //     bytes32[] memory commitmentHashes = preConfCommitmentStore.getCommitmentsByBlockNumber(blockNumber);
-    //     assertEq(commitmentHashes.length, 1);
-        
-    //     // Ensuring no rewards
-    //     assertEq(bidderRegistry.getProviderAmount(provider), 0);
-
-    //     // Detect slashing
-    //     uint256 postSlashStake = providerRegistry.checkStake(provider);
-    //     assertEq(postSlashStake + bid, ogStake);
-    //     assertEq(bidderRegistry.getAllowance(bidder), 250 ether);
-
-    // }
-
-    // // function test_ProcessCommitment_Slash() public {
-    // //   TODO(@ckartik): Add test
-    // // }
-
-    // function test_ProcessCommitment_Reward() public {
-
-    //      string memory txnHash = "0xkartik";
-    //      /*
-    //      Temporarily hardcoding the values for the following variables for future testing.
-    //     string
-    //         memory cHash = "0x31dca6c6fd15593559dabb9e25285f727fd33f07e17ec2e8da266706020034dc";
-    //     bytes
-    //         memory signature = "0xb170d082db1bf77fa0b589b9438444010dcb1e6dd326b661b02eb92abe4c066e243bb0d214b01667750ba2c53ff1ab445fd784b441dbc1f30280c379f002cc571c";
-    //     */
-    //     uint64 bid = 2;
-    //     uint64 blockNumber = 2;
-    //     bytes memory bidSignature = bytes(
-    //         hex"c10688ea554c1dae605619fa7f75103fb483ab6b5ad424e4e232f5da4449503a27ef6aed49b85bfd0e598650831c861a55a5eb197d9279d6a5667efaa46ab8831c"
-    //     );
-    //     bytes
-    //         memory commitmentSignature = hex"ff7e00cf5c2d0fa9ef7c5efdca68b285a664a3aab927eb779b464207f537551f4ff81b085acf78b58ecb8c96c9a4efcb2172a0287f5bf5819b49190f6e2d2d1e1b";
-    //     bytes32 commitmentIndex = preConfCommitmentStore.storeCommitment(bid, blockNumber, txnHash, bidSignature, commitmentSignature);
-
-    //     bool isSlash = false;
-    //     vm.expectEmit(true, false, false, true);
-    //     emit CommitmentProcessed(commitmentIndex, isSlash);
-    //     oracle.processCommitment(commitmentIndex, isSlash);
-    // }
-
-
     /**
     constructAndStoreCommitment is a helper function to construct and store a commitment
      */
```

Comment:
I am assuming this will change the abi and contract address if so please fix them in other repos and test to make sure it works.

Reply:
Yes it will, Though I don't think the onweronly aspect impacts the parts of the mev-commit iterface. I plan to test these before hand. It will change the contract address for example.